src/Mobs/Path.cpp

```diff
@@ -157,8 +157,14 @@ bool cPath::IsSolid(const Vector3i & a_Location)
 	int RelZ = a_Location.z - m_Chunk->GetPosZ() * cChunkDef::Width;
 
 	m_Chunk->GetBlockTypeMeta(RelX, a_Location.y, RelZ, BlockType, BlockMeta);
-	if ((BlockType == E_BLOCK_FENCE) || (BlockType == E_BLOCK_FENCE_GATE))
+	if (
+			(BlockType == E_BLOCK_FENCE) ||
+			(BlockType == E_BLOCK_FENCE_GATE) ||
+			(BlockType == E_BLOCK_NETHER_BRICK_FENCE) ||
+			((BlockType >= E_BLOCK_SPRUCE_FENCE_GATE) && (BlockType <= E_BLOCK_ACACIA_FENCE))
+		)
```

Comment:
Noticed after merge, sorry.
This code will change thoroughly anyways, so to avoid polluting the commit history, I'll leave it. It's not staying for long either way.